src/config.py

```diff
@@ -31,3 +38,87 @@ def is_benchmark_mode_enabled():
 
 def is_build_enabled():
     return parse_dlc_developer_configs("build", "do_build")
+
+
+def is_ec2_test_enabled():
+    return parse_dlc_developer_configs("test", "ec2_tests")
+
+
+def is_ecs_test_enabled():
+    return parse_dlc_developer_configs("test", "ecs_tests")
+
+
+def is_eks_test_enabled():
+    return parse_dlc_developer_configs("test", "eks_tests")
+
+
+def is_sanity_test_enabled():
+    return parse_dlc_developer_configs("test", "sanity_tests")
+
+
+def is_sm_local_test_enabled():
+    return parse_dlc_developer_configs("test", "sagemaker_local_tests")
+
+
+def are_efa_tests_enabled():
+    return parse_dlc_developer_configs("test", "efa_tests")
+
+
+def is_scheduler_enabled():
+    return parse_dlc_developer_configs("test", "use_scheduler")
+
+
+def get_allowed_sagemaker_remote_tests_config_values(disabled_only=False, enabled_only=False):
+    """
+    Retrieve allowed SM remote tests config values
+    """
+    disabled = ("off", "", False)
+    enabled = ("default", "release_candidate", "rc")
+    if disabled_only:
+        return disabled
+    if enabled_only:
+        return enabled
+    return enabled + disabled
+
+
+def get_sagemaker_remote_tests_config_value():
+    """
+    Get the actual config option for sm remote tests
+    """
+    sm_config_value = parse_dlc_developer_configs("test", "sagemaker_remote_tests")
+    allowed_config_values = get_allowed_sagemaker_remote_tests_config_values()
+    if sm_config_value not in allowed_config_values:
+        LOGGER.warning(
+            f"Unrecognized sagemaker_remote_tests config {sm_config_value}. "
+            f"Please ensure it is one of {allowed_config_values}, or your tests may not get triggered as expected."
+        )
+    return sm_config_value
+
+
+def is_sm_remote_test_enabled():
```

Comment:
Frankly seems overcomplicated. I like [this approach.](https://github.com/aws/deep-learning-containers/pull/1269/files#diff-5d5511865c124cd43c10b4a8f75f026de33c01632acdee7f0fbc849ae5498eb1R248)
Smthng like
`
if value not in ["default", "rc"]:
        LOGGER.warning("unrecognized SM remote test type. Disabling execution.")
        return false
return true
`